joint-discovery-gatk4.hg38.wgs.inputs.json

```diff
@@ -1,92 +1,96 @@
-  "JointDiscovery_GATK4.ApplyRecalibrationFilterForSNPs.java_opt": "-Xms4000m",
-  "JointDiscovery_GATK4.ApplyRecalibrationFilterForINDELs.java_opt": "-Xms4000m",
+  "JointGenotyping.SNPsVariantRecalibratorScattered.java_opt": "-Xmx3g -Xms3g",
+  "JointGenotyping.CollectMetricsOnFullVcf.java_opt": "-Xmx6g -Xms6g",
+  "JointGenotyping.IndelsVariantRecalibrator.java_opt": "-Xmx24g -Xms24g",
+  "JointGenotyping.HardFilterAndMakeSitesOnlyVcf.java_opt": "-Xmx3g -Xms3g",
+  "JointGenotyping.SNPGatherTranches.java_opt": "-Xmx6g -Xms6g",
+  "JointGenotyping.CollectMetricsSharded.java_opt": "-Xmx6g -Xms6g",
+  "JointGenotyping.SitesOnlyGatherVcf.java_opt": "-Xmx6g -Xms6g",
+  "JointGenotyping.ApplyRecalibration.java_opt": "-Xmx5g -Xms5g",
+  "JointGenotyping.FinalGatherVcf.java_opt": "-Xmx6g -Xms6g",
+  "JointGenotyping.ImportGVCFs.java_opt": "-Xmx4g -Xms4g",
+  "JointGenotyping.SNPsVariantRecalibratorCreateModel.java_opt": "-Xmx100g -Xms100g",
+  "JointGenotyping.GatherMetrics.java_opt": "-Xmx2g -Xms2g",
+  "JointGenotyping.GenotypeGVCFs.java_opt": "-Xmx5g -Xms5g",
 
   "##_COMMENT7": "MEMORY ALLOCATION", 
-  "JointDiscovery_GATK4.ImportGVCFs.mem_size": "7 GB",
-  "JointDiscovery_GATK4.GenotypeGVCFs.mem_size": "7 GB",
-  "JointDiscovery_GATK4.MergeVCFs.mem_size": "2 GB",
-  "JointDiscovery_GATK4.BuildVQSRModelForSNPs.mem_size": "5 GB",
-  "JointDiscovery_GATK4.BuildVQSRModelForINDELs.mem_size": "5 GB",
-  "JointDiscovery_GATK4.ApplyRecalibrationFilterForSNPs.mem_size": "5 GB",
-  "JointDiscovery_GATK4.ApplyRecalibrationFilterForINDELs.mem_size": "5 GB",
+  "JointGenotyping.CollectMetricsSharded.mem_size": "7 GB",
+  "JointGenotyping.ImportGVCFs.mem_size": "7 GB",
+  "JointGenotyping.IndelsVariantRecalibrator.mem_size": "26 GB",
+  "JointGenotyping.ApplyRecalibration.mem_size": "7 GB",
+  "JointGenotyping.CollectMetricsOnFullVcf.mem_size": "7 GB",
+  "JointGenotyping.GenotypeGVCFs.mem_size": "7 GB",
+  "JointGenotyping.FinalGatherVcf.mem_size": "7 GB",
+  "JointGenotyping.SitesOnlyGatherVcf.mem_size": "7 GB",
+  "JointGenotyping.SNPsVariantRecalibratorScattered.mem_size": "3.5 GB",
+  "JointGenotyping.SNPsVariantRecalibratorCreateModel.mem_size": "104 GB",
+  "JointGenotyping.DynamicallyCombineIntervals.mem_size": "3 GB",
+  "JointGenotyping.GatherMetrics.mem_size": "3 GB",
+  "JointGenotyping.HardFilterAndMakeSitesOnlyVcf.mem_size": "3.5 GB",
+  "JointGenotyping.SNPGatherTranches.mem_size": "7 GB",
 
   "##_COMMENT8": "DISK SIZE ALLOCATION",
-  "JointDiscovery_GATK4.ImportGVCFs.disk_size": 100,
-  "JointDiscovery_GATK4.GenotypeGVCFs.disk_size": 100,
-  "JointDiscovery_GATK4.MergeVCFs.disk_size": 100,
-  "JointDiscovery_GATK4.BuildVQSRModelForSNPs.disk_size": 100,
-  "JointDiscovery_GATK4.BuildVQSRModelForINDELs.disk_size": 100,
-  "JointDiscovery_GATK4.ApplyRecalibrationFilterForSNPs.disk_size": 100,
-  "JointDiscovery_GATK4.ApplyRecalibrationFilterForINDELs.disk_size": 100
+  "JointGenotyping.small_disk": 100,
```

Comment:
small_disk = (num_gvcfs / 10) + 10
medium_disk = (num_gvcfs * 15) + 10 
huge_disk = num_gvcfs + 10

small/medium/huge dont make as much sense as far as their names because this workflow doesnt use NIO

Reply:
Currently medium would be a larger disk size then huge, is medium and huge mistakenly switched because ?

Reply:
Medium was medium because we used to only grab a sliver of a gvcf for those calls with NIO.  With NIO removed the disk size for those calls has bloated to be bigger than our `huge_disk`.